docs/dev/Lua API.rst

```diff
@@ -5766,82 +5766,129 @@ The names of the functions are also available as the keys of the
 building-hacks
 ==============
 
-This plugin overwrites some methods in workshop df class so that mechanical workshops are possible. Although
-plugin export a function it's recommended to use lua decorated function.
+This plugin overwrites some methods in workshop df class to change and extend workshop
+functionality. See exported function list for details.
 
 .. contents::
   :local:
 
 Functions
 ---------
 
-``registerBuilding(table)`` where table must contain name, as a workshop raw name, the rest are optional:
+* ``setOwnableBuilding(workshop_type, enable)``
 
-    :name:
-        custom workshop id e.g. ``SOAPMAKER``
+  Set workshop to be included in zones (such as bedroom or inn).
 
-        .. note:: this is the only mandatory field.
+  :workshop_type:   custom workshop string id e.g. ``SOAPMAKER`` or numeric id
+  :enable:   boolean value to enable or disable this functionality
```

Comment:
can this be usefully dynamically toggled? what happens if a workshop type that *was* ownable is now not ownable? Does it make sense to have this parameter at all, or should this call just be to flip the value from false (the default) to true?

Reply:
yeah probably there is no reason to call this with false